.github/workflows/code_review.yml

```diff
@@ -12,9 +12,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: nianxiongdi/ai-codereviewer@ai
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          TOKEN:  ${{ secrets.TOKEN }}
```

Comment:
The environment variable `GITHUB_TOKEN` should be used instead of `TOKEN` to ensure proper authentication with GitHub actions.